<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teImageMap</class>
 <widget class="QWidget" name="teImageMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>609</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageMap Texture Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frClass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="lyTextureClass">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_84">
        <property name="text">
         <string>Texture class</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="textureClass"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frGray">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="lyGrayscale">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_85">
        <property name="text">
         <string>Grayscale channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="grayscaleChannel">
        <item>
         <property name="text">
          <string>Automatic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>From Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>From Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>From Blue</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="normalMap">
     <property name="text">
      <string>Normal map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ReImageMapManager" name="imTextureName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="textureFileName">
      <string>.../</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>6</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Re2DTextureMapping" name="textureMapping">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>frGray</zorder>
  <zorder>frClass</zorder>
  <zorder>imTextureName</zorder>
  <zorder>frame</zorder>
  <zorder>normalMap</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Re2DTextureMapping</class>
   <extends>QWidget</extends>
   <header>Re2DTextureMapping.h</header>
  </customwidget>
  <customwidget>
   <class>ReImageMapManager</class>
   <extends>QWidget</extends>
   <header>ReImageMapManager.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
